.eslintrc

```diff
@@ -0,0 +1 @@
+/home/estacio/.config/yarn/global/node_modules/vtex/.eslintrc
```

Comment:
`vtex local eslint` creates a symlink on your project
symlinks should not be committed 

I don't think it should be a symlink though, because `.eslintrc` should indeed be committed.

Reply:
This changes all the time :(

Reply:
Understood. I now committed the whole file.